packages/rich-text/src/helpers/__tests__/validations.test.ts

```diff
@@ -0,0 +1,52 @@
+import { createFakeFieldAPI } from '@contentful/field-editor-test-utils';
```

Comment:
@MayaGillilanContentful testing the logic for validating marks in these unit tests